schemachange/sc_alter_table.c

```diff
@@ -548,6 +591,11 @@ int do_alter_table(struct ireq *iq, struct schema_change_type *s,
         return rc;
     }
     newdb->iq = NULL;
+
+    /* check for rename outside of taking schema lock */
```

Comment:
This comment describes what the following function is doing, right? Can we move it before the function itself?

Reply:
You mean move the comment to the function definition? I moved this out from finalize_alter_table to here, do you think this is correct? I did this because I now grab schemalk early in `osql_process_packet` in case `OSQL_DONE` and only release schemalk after the transaction commits/aborts.